logic/src/commonMain/kotlin/com/wire/kalium/logic/data/call/mapper/CallMapper.kt

```diff
@@ -144,14 +145,15 @@ class CallMapperImpl : CallMapper {
         CallStatus.CLOSED -> CallEntity.Status.CLOSED
     }
 
-    private fun toCallStatus(callStatus: CallEntity.Status): CallStatus = when (callStatus) {
+    override fun toCallStatus(callStatus: CallEntity.Status?): CallStatus = when (callStatus) {
         CallEntity.Status.STARTED -> CallStatus.STARTED
         CallEntity.Status.INCOMING -> CallStatus.INCOMING
         CallEntity.Status.MISSED -> CallStatus.MISSED
         CallEntity.Status.ANSWERED -> CallStatus.ANSWERED
         CallEntity.Status.ESTABLISHED -> CallStatus.ESTABLISHED
         CallEntity.Status.STILL_ONGOING -> CallStatus.STILL_ONGOING
         CallEntity.Status.CLOSED -> CallStatus.CLOSED
+        else -> CallStatus.CLOSED
```

Comment:
I think we can have another function to map the call status we receive from the Conversation List, such as:

```
fun toCallStatusFromConversationList(callStatus: CallEntity.Status?): CallStatus = when (callStatus) {
  null -> CallStatus.CLOSED
  else -> toCallStatus(callStatus = callStatus)
}
```

This way we keep the originality of the current `toCallStatus` mapping function and not have a default else to `CallStatus.CLOSED` as it could bring future problems as @trOnk12 mentioned.